src/query/plan/operator.cpp

```diff
@@ -321,6 +329,13 @@ bool CreateExpand::CreateExpandCursor::Pull(Frame &frame, ExecutionContext &cont
 
   if (!input_cursor_->Pull(frame, context)) return false;
 
+  if (context.auth_checker && !(context.auth_checker->Accept(*context.db_accessor, self_.edge_info_.edge_type,
+                                                             memgraph::auth::FineGrainedPermission::CREATE_DELETE) &&
+                                context.auth_checker->Accept(*context.db_accessor, self_.node_info_.labels,
+                                                             memgraph::auth::FineGrainedPermission::CREATE_DELETE))) {
+    spdlog::error("Error while creating an edge due to not having enough permission!");
+    return false;
+  }
```

Comment:
Same here as for create? Isn't this enough once when we know the labels and edge type?